static/lang/re-en.json

```diff
@@ -2569,7 +2569,14 @@
                 "Wit": "The foe takes a -2 circumstance penalty to attack rolls against you until the start of your next turn. @UUID[Compendium.pf2e.feat-effects.Item.cu10OjQUdY0a9gpj]{Effect: Exemplary Finisher (Wit)}"
             },
             "ExorcistDedication": {
-                "SpiritDwelling": "Your Spirit Dwelling contains a spirit"
+                "SpiritDwelling": "Your Spirit Dwelling contains a spirit",
+                "SpiritsMercy": {
+                    "Prompt": "Select the cost of your Spirit's Mercy",
+                    "PromptLevel": "Select the level of your Spirit Remnant",
+                    "ResistanceLabel": "spirits"
```

Comment:
Move this to the custom IWR localizations

Reply:
Moved!